app/assets/stylesheets/shared/header-subnav.scss

```diff
@@ -0,0 +1,44 @@
+//
+// Header subnav styles
+//
+
+.prx-app-bar {
+  border-bottom: 1px solid $gray-300;
+}
+
+.prx-app-nav {
+  // display: flex;
+  // flex-direction: row;
+  // justify-content: flex-start;
+  // align-items: center;
+  // margin: 0;
+  // padding: 0;
+
+  // .navbar-nav {
+  //   display: flex;
+  //   flex-direction: row;
+  //   flex-wrap: wrap;
+  //   margin: 0;
+  //   padding: 0;
+  // }
```

Comment:
Ah! Can you remove the commented out lines? I imagine this is because you were making it the styles were represented elsewhere. 👍